lib/hanami/action/application_action.rb

```diff
@@ -43,6 +48,17 @@ def resolve_view_context
         end
       end
 
+      def resolve_paired_view(action_class)
+        view_identifiers = application.config.actions.view_name_inferrer.(
+          action_name: action_class.name,
+          provider: provider
+        )
+
+        view_identifiers.each_with_object(nil) { |identifier|
```

Comment:
@timriley very very minor thing: why this and not `view_identifiers.find`? What am I missing?

Reply:
@jodosha You're right! This could with with `#find`/`#detect` too. I'm going to switch it over.